plugins/frontend/backstage-plugin-github-pull-requests/src/components/usePullRequestsStatistics.ts

```diff
@@ -41,7 +41,7 @@ export type PullRequestStatsCount = {
 };
 export type PullRequestStatsData = {
   createdAt: string;
-  closedAt: string;
+  closedAt: string | null;
```

Comment:
Why null and not undefined?

Reply:
The new octokit returns the closed_at field as string | null.